src/arrays/arraymath.jl

```diff
@@ -59,7 +59,7 @@ function *{B<:OrthonormalBasis}(qm1::QuMatrix{B}, qm2::QuMatrix{B})
                    bases(qm2,2))
 end
 
-*(dm1::DualMatrix, dm2::DualMatrix) = (dm1.qarr*dm2.qarr)'
+*(dm1::DualMatrix, dm2::DualMatrix) = QuArray(coeffs(dm1)*coeffs(dm2),bases(dm1,1), bases(dm2,2))
```

Comment:
The removed version is also more efficient, AFAIK, since it doesn't have to take the conjugate transpose. 

But I suppose you're right about having an accessor function for the `qarr` field. The problem is `qarr` works really well as a descriptive, concise variable name, and I'd like to keep it if possible, so I wonder what to call the accessor function. Also, technically, `ctranspose` is basically implemented as an accessor function. But I suppose we should have an ACTUAL one, with which we can redefine `ctranspose` and all the other instances of direct field access. 

@amitjamadagni You said you were getting inconsistent behavior with the old definition; do you have an example?